packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -1017,19 +1005,17 @@ export class DbClientMetadata {
   static keyPath = 'clientId';
 
   constructor(
+    // Note: Previous schema versions included a field
+    // "lastProcessedDocumentChangeId". Don't use anymore.
```

Comment:
nit: Might be better to keep this note at the bottom of the list, since that's where we would be adding new fields...